server/src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -406,27 +419,72 @@ else if (type.isType(ConsumerTypeEnum.HYPERVISOR)) {
         return promotedContent;
     }
 
+    /**
+     * Fetches the path to use for content used to generate the ENTITLEMENT_DATA certificate
+     * extension
+     *
+     * @param owner
+     *  the owner of the entitlement for which the content path is being generated
+     *
+     * @param environment
+     *  the environment to which the consumer receiving the entitlement belongs
+     *
+     * @return
+     *  the container content path for the given owner and environment
+     */
+    private String getContainerContentPath(Owner owner, Environment environment) throws IOException {
+        // Fix for BZ 1866525:
+        // - In hosted, SCA entitlement content path needs to use a dummy value to prevent existing
+        //   clients from breaking, while still being clear the path is present for SCA
```

Comment:
Just curious why/who having a dummy value prevents existing clients from breaking?

Reply:
The big one is subman/rhsmlib's certificate2.py doesn't check that the extension properly exists before attempting to use the value, so the result you get is an exception rather than a false from ```check_paths```, or empty array of paths in ```provided_paths```.